File: Modul-6/Soal-1/PRAK601-2310817110013-Randy_Febrian.py
import numpy as np

x, y = map(int, input().split())

element = np.array(list(map(int, input().split())))
matriks = element.reshape((x, y))

for i in range(x):
    for j in range(y):
        print(matriks[i][j], end=' ')
    print()

File: Modul-6/Soal-1/PRAK601-2310817110013-Randy_Febrian.c
#include <stdio.h>

int main(){
    int x, y;
    scanf("%d %d", &x, &y);

    int matriks[x][y];
    for (int i = 0; i < x; i++) {
        for (int j = 0; j < y; j++) {
            scanf("%d", &matriks[i][j]);
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}